src/Mvc/Mvc.ApiExplorer/test/EndpointMetadataApiDescriptionProviderTest.cs

```diff
@@ -512,7 +513,7 @@ public void TestParameterIsRequiredForObliviousNullabilityContext()
     [Fact]
     public void TestParameterAttributesCanBeInspected()
     {
-        var apiDescription = GetApiDescription(([Description("The name.")] string name) => { });
+        var apiDescription = GetApiDescription(([System.ComponentModel.Description("The name.")] string name) => { });
```

Comment:
Oh wow, I didn't realize MVC already supported the ComponentModel attribute. Perhaps that might be a reason to reuse it? Saving the planet by re-using one attribute at a time.

Reply:
Yeah, I called this stomping out in my email as a possible issue. Particularly when you account for the issues with name stomping mentioned above.

The downside of reusing this attribute for descriptions is that there's not a similar attribute for summaries so there would be this weird incongruence where we used an attribute from one namespace for descriptions and another for summaries.